plio/urls.py

```diff
@@ -58,6 +64,7 @@
 api_router.register(r"session-answers", SessionAnswerViewSet)
 api_router.register(r"events", EventViewSet)
 api_router.register(r"tags", TagViewSet)
+api_router.register(r"organization-user", OrganizationUserViewSet)
```

Comment:
everything else is plural
```suggestion
api_router.register(r"organization-users", OrganizationUserViewSet)
```